processing/src/main/java/io/druid/query/aggregation/NullableAggregator.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Licensed to Metamarkets Group Inc. (Metamarkets) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. Metamarkets licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.druid.query.aggregation;
+
+import io.druid.segment.ColumnValueSelector;
+
+public class NullableAggregator implements Aggregator
+{
+  private final Aggregator delegate;
+  private final ColumnValueSelector selector;
+  private boolean isNull;
```

Comment:
Need to set this to `true` thought.

Reply:
this is set to true in reset() method. Called it from constructor too.

Reply:
I don't think it is true for all the aggregators, for instance, io.druid.query.aggregation.CountAggregator. It is better to initialized since it should be true, not sure why you want to avoid this?

Reply:
done.